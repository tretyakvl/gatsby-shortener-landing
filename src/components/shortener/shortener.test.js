import React from 'react'
import renderer from 'react-test-renderer'
import Shortener from './shortener'

describe('Shortener', () => {
  it('renders correctly with zero links', () => {
    const tree = renderer
      .create(<Shortener />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
