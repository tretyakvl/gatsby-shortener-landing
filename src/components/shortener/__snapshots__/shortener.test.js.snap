// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shortener renders correctly with zero links 1`] = `
<section
  className="shortener"
>
  <div
    className="shortener__container"
    style={
      Object {
        "opacity": 0,
        "transform": "translateX(-20px) translateZ(0)",
      }
    }
  >
    <form
      className="form"
      onSubmit={[Function]}
    >
      <div
        className="form__notificator "
      >
        <input
          id="shortener-url"
          onChange={[Function]}
          placeholder="Shorten a link here..."
          type="text"
          value=""
        />
        <span>
          
        </span>
      </div>
      <button
        className="button-module button square thin500"
      >
        Shorten it!
      </button>
    </form>
    <ul
      className="shortener__list"
    />
  </div>
</section>
`;
